test/unit/Axes.spec.js

```diff
@@ -349,129 +349,133 @@ describe("Axes", function () {
-			if (this.inst) {
-				this.inst.destroy();
-				this.inst = null;
-			}
-			if (this.input) {
-				this.input.destroy();
-				this.input = null;
-			}
-			this.releaseHandler.resetHistory();
-			this.finishHandler.resetHistory();
-			cleanup();
-		});
-		it("should check release event occurs when swiping on ios safari edge. (left to right)", (done) => {
-			// Given, When
-			// clientX + => -
-			Simulator.gestures.pan(this.el, {
-				pos: [30, 30],
-				deltaX: -50,
-				deltaY: 10,
-				duration: 200,
-				easing: "linear"
-			}, () => {
-				// Then
-				// for test animation event
-				setTimeout(() => {
-					const releaseEvent = this.releaseHandler.getCall(0).args[0];
-					expect(this.releaseHandler.calledOnce).to.be.true;
-					expect(releaseEvent.inputEvent.isFinal).to.be.false;
-					expect(releaseEvent.isTrusted).to.be.true;
+				if (this.input) {
+					this.input.destroy();
+					this.input = null;
+				}
+				this.releaseHandler.resetHistory();
+				this.finishHandler.resetHistory();
+				cleanup();
+			});
+			it("should check release event occurs when swiping on ios safari edge. (left to right)", (done) => {
+				// Given, When
+				// clientX + => -
+				Simulator.gestures.pan(this.el, {
+					pos: [30, 30],
+					deltaX: -50,
+					deltaY: 10,
+					duration: 200,
+					easing: "linear"
+				}, () => {
+					// Then
+					// for test animation event
+					setTimeout(() => {
+						const releaseEvent = this.releaseHandler.getCall(0).args[0];
+						expect(this.releaseHandler.calledOnce).to.be.true;
+						expect(releaseEvent.inputEvent.isFinal).to.be.false;
+						expect(releaseEvent.isTrusted).to.be.true;
 
-					const finishEvent = this.finishHandler.getCall(0).args[0];
-					expect(this.finishHandler.called).to.be.true;
-					expect(finishEvent.isTrusted).to.be.true;
-					done();
-				}, 500);
+						const finishEvent = this.finishHandler.getCall(0).args[0];
+						expect(this.finishHandler.called).to.be.true;
+						expect(finishEvent.isTrusted).to.be.true;
+						done();
+					}, 500);
+				});
 			});
-		});
-		it("should check release event occurs when swiping a little on ios safari edge. (right to left)", (done) => {
-			// Given, When
-			// window.innerWidth => window.innerWidth - 15
-			Simulator.gestures.pan(this.el, {
-				pos: [window.innerWidth, 30],
-				deltaX: -15,
-				deltaY: 0,
-				duration: 200,
-				easing: "linear"
-			}, () => {
-				// Then
-				// for test animation event
-				setTimeout(() => {
-					const releaseEvent = this.releaseHandler.getCall(0).args[0];
-					expect(this.releaseHandler.calledOnce).to.be.true;
-					expect(releaseEvent.inputEvent.isFinal).to.be.false;
-					// expect(releaseEvent.isTrusted).to.be.true;
+			it("should check release event occurs when swiping a little on ios safari edge. (right to left)", (done) => {
+				// Given, When
+				// window.innerWidth => window.innerWidth - 15
+				Simulator.gestures.pan(this.el, {
+					pos: [window.innerWidth, 30],
```

Comment:
I think it would be nicer if you set the start pos to `window.innerWidth - iOSEdgeSwipeThreshold`